UnitsNet.Tests/CustomCode/VolumeTests.cs

```diff
@@ -75,8 +75,12 @@ public class VolumeTests : VolumeTestsBase
 
         protected override double TeaspoonsInOneCubicMeter => 202884.13621105801;
 
+        protected override double ImperialBeerBarrelsInOneCubicMeter => 6.1102568972;
+
         protected override double UkTablespoonsInOneCubicMeter => 66666.6666667;
 
+        protected override double UsBeerBarrelsInOneCubicMeter => 8.5216790723083;
```

Comment:
What is the source of this constant?

From [wiki](https://en.wikipedia.org/wiki/Barrel_(unit)#Fluid_barrel_in_the_US_and_UK) I found:
> In the US most fluid barrels (apart from oil) are 31.5 US gallons (26 imp gal; 119 L) (half a hogshead), but a beer barrel is 31 US gallons (26 imp gal; 117 L).

I read that as `1 US beer barrel = 117 L = 0.117 m^3 => 1 m^3 = 8.54700854700.. US beer barrels`.

Reply:
Hmmm, google and https://www.convertunits.com/from/US+gallons/to/barrel+[US,+beer] say 31 US Gal. Convert-to.com also says 31: http://convert-to.com/conversion/volume/convert-us-barrel-beers-to-gal-us.html.

So I then converted 31 US Gal to m^3 (8.5 216 791)

Reply:
Aha, 31 US gallons = 117.348 L.
It seems 31 US gallons is the truth then and 117 L was just a rounded value. Should be good as-is then.